reposilite-backend/src/main/kotlin/com/reposilite/settings/api/LocalConfiguration.kt

```diff
@@ -74,6 +74,19 @@ class LocalConfiguration : Serializable {
         lateinit var password: String
     }
 
+    /* Webhooks */
+    @Description("")
+    @Description("# Webhook Service")
+    @Description("Options: slack, discord, off")
+    val webhookService = reference("off")
+
+    @Description("# Webhook URL")
+    val webhookUrl = reference("https://webhook.example.com/v1/channel/id")
+
+    @Description("# Webhook Type")
+    @Description("# Only for Discord at the moment. Options: embed, message")
+    val webhookType = reference("embed")
+
```

Comment:
It should be moved to shared configuration. Let's ignore this until the following PR will be merged:
* #1081